components/chat.tsx

```diff
@@ -185,6 +187,7 @@ const Chat: React.FC<ScriptProps> = ({
                 disableInput={
                   disableInput || !running || waitingForUserResponse
                 }
+                noChat={noChat}
```

Comment:
If the answer to the other question is "they are the same" then I think this can be
```suggestion
                noChat={disableInput}
```